CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [5.5.1] - 2020-03-30
+
+### Added
+
+- Enabled per-cluster configuration of kube-proxy's `conntrackMaxPerCore` parameter.
+
+### Changed
+
+- Streamlined image templating for core components for quicker and easier releases in the future.
+- Retrieve component versions from releases instead.
```

Comment:
Instead of...

Reply:
I just changed the whole sentence because I didn't know how to phrase it :man_shrugging:

Reply:
Yeah fair enough